preprocess-and-train.py

```diff
@@ -2,15 +2,19 @@
 import cv2
 from model import generate_model
 
-TRAINING_DATA_DIR = "training_data"
+TRAINING_DATA_DIR = "C:\\Users\\vijay\\PycharmProjects\\opencv_begin\\venv\\training_data"
 
 # here write code to pre process the images as required
 # but make sure to have a copy of the original training data
 # or that you write your code in such a way that you pre-process on the fly
 # pre-processing on the fly is possible because this is small project and
 # not a lot of images will be there as well.
 def preprocess(img):
-    # your code goes here
+    width = 225
+    height = 225
+    dimensions = (width, height)
+    img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+    img = cv2.resize(img, dimensions, interpolation=cv2.INTER_CUBIC)
```

Comment:
what other preprocessing techniques did you try?
and what were the results of those techniques? @vijay1827

Reply:
Using cv2.COLOR_BGR2HSV color space accuracy is 97.75
AND using cv2.COLOR_BGR2GRAY color space accuracy is 96.7

Reply:
using cv2.COLOR_BGR2RGB color space accuracy is 92.00

Reply:
using GaussianBlur accuracy 98